src/main/java/org/jenkinsci/test/acceptance/po/ToolInstallation.java

```diff
@@ -77,6 +77,12 @@ private static boolean hasUpdatesFor(final Jenkins jenkins, Class<? extends Tool
         final ConfigurablePageObject page = ensureConfigPage(jenkins);
 
         final String name = type.getAnnotation(ToolInstallationPageObject.class).name();
+        final Control expandButton = page.control(by.button(name + " installations..."));
+        try {
+            expandButton.click();
+        } catch (Exception e) {
```

Comment:
Be more specific which exception type is safe to ignore here.

Reply:
Done